src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  margin: 10vh;
+  // flex-flow: space-around;
+  // align-items: center;
+`;
+
+const Buttons = styled.button`
+  font-size: 5vh;
+  margin: 0 1vw;
+`;
+
+export default function Button({ grid, nextMove, changeAllMoves }) {
+  // let text = "Start";
+  const [text, setText] = useState("Start");
```

Comment:
I think changing the 'text' variable name to something more descriptive like 'startOrStop' or 'startOrStopText'  would make it easier to read